modules/appeals_api/spec/services/appeals_api/pdf_construction/notice_of_disagreement/v1/form_data_spec.rb

```diff
@@ -105,16 +105,6 @@ module V1
             it { expect(form_data.contestable_issues).to eq(notice_of_disagreement.form_data.dig('included')) }
           end
 
-          describe '#stamp_text' do
```

Comment:
Even if we removed `#stamp_text` we should still move this test somewhere instead of removing it outright, IMO.

Reply:
Yeah. I tried to move it but was unexpectedly stumped. Still working on it.